x/staking/types/validator.go

```diff
@@ -467,7 +467,7 @@ func (v Validator) PotentialTendermintPower() int64 {
 var _ sdk.Validator = Validator{}
 
 // nolint - for sdk.Validator
-func (v Validator) GetJailed() bool               { return v.Jailed }
+func (v Validator) IsJailed() bool                { return v.Jailed }
```

Comment:
I don't understand why we need getters for all these public members 🤔

Reply:
For implementing the interface Validator ?

```go
type Validator interface {
	IsJailed() bool                                 // whether the validator is jailed
	GetMoniker() string                             // moniker of the validator
	GetStatus() BondStatus                          // status of the validator
	GetOperator() ValAddress                        // operator address to receive/return validators coins
	GetConsPubKey() crypto.PubKey                   // validation consensus pubkey
	GetConsAddr() ConsAddress                       // validation consensus address
	GetTokens() Int                                 // validation tokens
	GetBondedTokens() Int                           // validator bonded tokens
	GetTendermintPower() int64                      // validation power in tendermint
	GetCommission() Dec                             // validator commission rate
	GetMinSelfDelegation() Int                      // validator minimum self delegation
	GetDelegatorShares() Dec                        // total outstanding delegator shares
	TokensFromShares(Dec) Dec                       // token worth of provided delegator shares
	TokensFromSharesTruncated(Dec) Dec              // token worth of provided delegator shares, truncated
	SharesFromTokens(amt Int) (Dec, Error)          // shares worth of delegator's bond
	SharesFromTokensTruncated(amt Int) (Dec, Error) // truncated shares worth of delegator's bond
}
```

Reply:
Yes it's because we have an interface 👍

Reply:
Fair enough